### PR TITLE
Feature/languages

### LINKED ISSUE #178: expose language picker in settings
we will have translations for Korean, Japanese, and Simplified Chinese. FR and ES are not translated, so those should be left out. I'm guessing the .ts files need to be remove from the locales folder?

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
Closes #178

### CODE REVIEW

File: src/features/moreTab/more/MoreScreen.tsx [hunk 1]
```diff
@@ -128,6 +129,13 @@ const MoreScreen = () => {
     )
   }, [t, dispatch])
 
+  const handleLanguageChange = useCallback(
+    (lng: string) => {
```
Comment (allenan) [inline src/features/moreTab/more/MoreScreen.tsx:133]: this is sort of silly but maybe we could name this `lang` or `language`? we use `lng` pretty extensively to mean `longitude` elsewhere :)

File: src/locales/ja.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
 export default {
-  back: 'Retour',
+  back: 'jBack',
```
Comment (allenan) [inline src/locales/ja.ts:2]: is this change intentional?

File: src/features/moreTab/more/MoreScreen.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/moreTab/more/MoreScreen.tsx:133]: yeah, I was thinking the same thing, but i18n uses `lng` so I stuck with it. I'll change it up.

File: src/locales/ja.ts (hunk 2 above)
Comment (matthewcarlreetz) [reply src/locales/ja.ts:2]: I mean, sort of 🤷 

I renamed the fr file to ja and just threw that in there for the moment. Should I copy over the English strings for now?

Comment (allenan) [reply src/locales/ja.ts:2]: oh i see. i guess i would delete anything that's not currently being used and let it fall back to the english translation

Comment (matthewcarlreetz) [reply src/locales/ja.ts]: I removed the key for now

**Comment by allenan** [comment]
is there a setting (maybe just in release builds?) to have strings without a translation in a given language fallback to english, rather than the i18n string?

image

**Review by allenan — APPROVED** [review]
works really well!
